horace_core/spaghetti_plot.m

```diff
@@ -13,10 +13,11 @@
 %   >> spaghetti_plot(...,'logscale')              % plots intensity in log10 scale
 %   >> spaghetti_plot(...,'clim',[cmin cmax])      % sets the colorscale (needed for logscale)
 %
-%   >> wdisp=spaghetti_plot(...)                   % outputs the cuts as a d2d array
-%   >> wdisp=spaghetti_plot(...,'noplot')          % outputs arrays without plotting
+%   >> wdisp = spaghetti_plot(...)                   % outputs the cuts as a d2d array
+%   >> wdisp = spaghetti_plot(...,'noplot')          % outputs arrays without plotting
 %
-%   >> [wdisp,cuts]=spaghetti_plot(...)            % generates a set of 1D cuts
+%   >> [wdisp,cuts] = spaghetti_plot(...)             % generates a set of 1D cuts
+%   >> [wdisp, cuts] = spaghetti_plot(...,'withpix')  % return cuts as sqw rather than d1ds
```

Comment:
So the `'withpix'` option only applies to the output cuts? (The output slices `wdisp` will always be `d2d`?)

Issue [#1362](https://github.com/pace-neutrons/Horace/issues/1362) seems to suggest that if `'withpix'` is given that the output slices `wdisp` will also be an array of 2D `sqw` objects? (But I don't know exactly what Christian wanted...)

Reply:
Should be sorted now in the latest.